expertise/service/expertise.py

```diff
@@ -825,15 +821,19 @@ def set_client_v2(self, client_v2):
         self.client_v2 = client_v2
         self.cloud.set_client(client_v2)
 
-    def compute_machine_type(self, client, client_v2, api_request):
-        config, _ = self._prepare_config(deepcopy(api_request), client_v1=client, client=client_v2)
+    def compute_machine_type(self, client_v1, client, job_id):
+        config, _ = self._validate_request(
```

Comment:
You don't need to call this function to get the config, you can call to `self.redis.load_job(job_id, get_user_id(client))` directly.